frame/society/src/lib.rs

```diff
@@ -1444,7 +1474,11 @@ impl<T: Config<I>, I: Instance> Module<T, I> {
 				members.sort();
 				// NOTE: This may cause member length to surpass `MaxMembers`, but results in no consensus
 				// critical issues or side-effects. This is auto-correcting as members fall out of society.
```

Comment:
in which contains this will happens, will this print `warn` regularly when MaxMembers is surpassed ?